Zend/zend_alloc.c

```diff
@@ -396,8 +396,7 @@ static ZEND_COLD ZEND_NORETURN void zend_mm_safe_error(zend_mm_heap *heap,
 }
 
 #ifdef _WIN32
-void
-stderr_last_error(char *msg)
+static void stderr_last_error(char *msg)
```

Comment:
nit: I believe `msg` ought to be const.

Reply:
Your nit is unrelated to the change I propose. If you believe `const` should be added, submit a PR for that.

Reply:
No need to be aggressive in the reply.

Reply:
Huh, aggressive, do you mean me? I wasn't. It was a reply to somebody who had another (unrelated) good idea to improve this piece of code, and I encouraged him to submit a PR, just like I do every time I have an idea to improve code. I think that's reasonable, and that being aggressive is your interpretation which I cannot follow.